src/cli/runCli.test.ts

```diff
@@ -1,9 +1,13 @@
 import { describe, expect, it, vi } from "vitest";
 
-import { version } from "../../package.json";
+// Used to be "import { version } ..", but bug in TypeScript 5.7:
+// https://github.com/microsoft/TypeScript/issues/60589#issuecomment-2501614830
+import packageConfig from "../../package.json" with { type: "json" };
```

Comment:
I don't think a bug in TypeScript. The repo isn't on Node16 resolution:

https://github.com/JoshuaKGoldberg/TypeStat/blob/6bfc04441f72cafbe4625cd4be0c610dfdff493e/tsconfig.json#L6-L7

What errors are you seeing? I don't have any issues running `npx typestat` locally.

Reply:
You can see the error in the CI logs for the type checking job if you click the red cross on the renovatebot commit in this PR:
```
Error: src/cli/runCli.test.ts(3,10): error TS1544: Named imports from a JSON file into an ECMAScript module are not allowed when 'module' is set to 'NodeNext'.
Error: src/cli/runCli.test.ts(3,25): error TS1543: Importing a JSON file into an ECMAScript module requires a 'type: "json"' import attribute when 'module' is set to 'NodeNext'.
```

Reply:
🤔 but type checking on `main` is passing, https://github.com/JoshuaKGoldberg/TypeStat/actions/runs/12097459370/job/33732857242

Reply:
Yeah, because main uses TypeScript 5.4.3, but this PR is on top of #1725 which bumps TypeScript to 5.7.2.

Reply:
Oh gotcha! Thanks, I'd missed that.

I need to context switch away from TypeStat for the next day or two, but will take a look when I can come back. 👍

Reply:
Sure, I'm not in a hurry with this, I just figured eventually you would need to upgrade to TypeScript 5.7.

Reply:
Ok coming back: it's intentional that we can't used named specifies with `.json` imports. This is TypeScript more correctly adhering to ECMAScript Modules. 🚀